lib/items.ts

```diff
@@ -2,45 +2,48 @@ import { Item, PlayedItem } from "../types/item";
 import { createWikimediaImage } from "./image";
 
 export function getRandomItem(deck: Item[], played: Item[]): Item {
-  const playedYears = played.map((item): number => {
-    return item.year;
-  });
-
   const periods: [number, number][] = [
-    [-100000, 1400],
-    [1400, 1850],
-    [1850, 1930],
-    [1930, 2020],
+    [-100000, 1000],
+    [1000, 1800],
+    [1800, 2020],
   ];
   const [fromYear, toYear] =
     periods[Math.floor(Math.random() * periods.length)];
   const avoidPeople = Math.random() > 0.5;
+  let distance = 5;
+  if (played.length < 11) { 
+    distance = 110 - 10 * played.length;
+  } else if (played.length > 40) {
+    distance = 1;
+  }
```

Comment:
This logic could be moved inside the `tooClose` function.

Reply:
This also increases the computational load, as the system is determining distance for every card in the deck, but if we're not worried about it for #22, we don't need to worry about it here.

Reverted.